internal/rego/rego.go

```diff
@@ -241,9 +248,54 @@ func (r *Rego) parseAnnotations(annotations *ast.Annotations) error {
 		r.enforcement = e
 	}
 
+	metaAnnotations, ok := annotations.Custom[annoAnnotations]
+	if ok {
+		a, ok := metaAnnotations.(map[string]interface{})
+		if !ok {
+			return fmt.Errorf("supplied annotations value is not a map[string]interface{}: %T", metaAnnotations)
+		}
+		if r.metaData == nil {
+			r.metaData = &MetaData{}
+		}
+		ans, err := switchToMap(a)
+		if err != nil {
+			return err
+		}
+		r.metaData.Annotations = ans
+	}
+
+	metaLabels, ok := annotations.Custom[annoLabels]
+	if ok {
+		l, ok := metaLabels.(map[string]interface{})
+		if !ok {
+			return fmt.Errorf("supplied labels value is not a map[string]interface{}: %T", metaLabels)
+		}
+		if r.metaData == nil {
+			r.metaData = &MetaData{}
+		}
+		labels, err := switchToMap(l)
```

Comment:
I think this is redundant, you can cast to `map[string]string` directly which is the only allowed type for K8s labels.

Reply:
Sorry for the late reply.
I am not sure if I get you right: 
When I directly use `metaAnnotations.(map[string]string)` instead of `metaAnnotations.(map[string]interface{})` the type check fails. Somehow I get a `map[string]interface{}` instead of a `map[string]string`

Reply:
OK, thank you for checking.